File: GGAndroid_2.0/GGDemo/src/main/java/io/ganguo/demo/viewmodel/page/EmptyDataViewModel.java
package io.ganguo.demo.viewmodel.page;

import android.view.View;

import io.ganguo.demo.R;
import io.ganguo.demo.databinding.PageEmptyDataBinding;
import io.ganguo.library.ui.view.ViewInterface;
import io.ganguo.vmodel.BaseViewModel;

/**
 * 空数据提示页面
 * Created by leo on 2018/11/10.
 */
public class EmptyDataViewModel extends BaseViewModel<ViewInterface<PageEmptyDataBinding>> {
    @Override
    public void onViewAttached(View view) {

    }

    @Override
    public int getItemLayoutId() {
        return R.layout.page_empty_data;
    }
}

File: GGAndroid_3.0/main/app-mvvm-v1-sample/src/main/java/io/ganguo/demo/viewmodel/page/EmptyDataViewModel.kt
package io.ganguo.demo.viewmodel.page

import android.view.View

import io.ganguo.demo.R
import io.ganguo.demo.databinding.PageEmptyDataBinding
import io.ganguo.viewmodel.core.viewinterface.ViewInterface
import io.ganguo.viewmodel.core.BaseViewModel

/**
 * 空数据提示页面
 * Created by leo on 2018/11/10.
 */
class EmptyDataViewModel : BaseViewModel<ViewInterface<PageEmptyDataBinding>>() {
    override fun onViewAttached(view: View) {

    }

    override val layoutId: Int by lazy { R.layout.page_empty_data }
}
